lighttpd: PKG_CONFIG_DEPENDS missing

Maintainer: @MikePetullo 
Environment: Kirkwood, LEDE HEAD.

Toggling `CONFIG_PACKAGE_lighttpd-mod-authn_gssapi` from `n` to `y` in an existing build gave rise to

```
make[3]: Entering directory '/tank/openwrt/scratch/builder-webserver/feeds/packages/net/lighttpd'
mkdir -p /tank/openwrt/scratch/builder-webserver/staging_dir/target-arm_xscale_musl_eabi/root-kirkwood/stamp
SHELL= flock /tank/openwrt/scratch/builder-webserver/tmp/.root-copy.flock -c 'cp -fpR /tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/.pkgdir/lighttpd/. /tank/openwrt/scratch/builder-webserver/staging_dir/target-arm_xscale_musl_eabi/root-kirkwood/'
touch /tank/openwrt/scratch/builder-webserver/staging_dir/target-arm_xscale_musl_eabi/root-kirkwood/stamp/.lighttpd_installed
mkdir -p /tank/openwrt/scratch/builder-webserver/bin/targets/kirkwood/generic/packages /tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/ipkg-arm_xscale/lighttpd-mod-authn_gssapi/CONTROL /tank/openwrt/scratch/builder-webserver/staging_dir/target-arm_xscale_musl_eabi/pkginfo
install -d -m0755 /tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/ipkg-arm_xscale/lighttpd-mod-authn_gssapi/usr/lib/lighttpd
cp -fpR /tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/ipkg-install/usr/lib/lighttpd/mod_authn_gssapi.so /tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/ipkg-arm_xscale/lighttpd-mod-authn_gssapi/usr/lib/lighttpd
cp: cannot stat '/tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45/ipkg-install/usr/lib/lighttpd/mod_authn_gssapi.so': No such file or directory
Makefile:197: recipe for target '/tank/openwrt/scratch/builder-webserver/bin/packages/arm_xscale/packages/lighttpd-mod-authn_gssapi_1.4.45-3_arm_xscale.ipk' failed
```

Removing `/tank/openwrt/scratch/builder-webserver/build_dir/target-arm_xscale_musl_eabi/lighttpd-1.4.45` and rebuilding produced success.  I believe this because the lighttpd makefile neglects to define `PKG_CONFIG_DEPENDS`.  I believe, but am not sure, that it would suffice to add `PKG_CONFIG_DEPENDS+=lighttpd-mod-$(1)` to the `BuildPlugin` macro's definition.

--- a/net/lighttpd/Makefile
+++ b/net/lighttpd/Makefile
@@ -21,7 +21,9 @@ PKG_LICENSE_FILES:=COPYING
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
-PKG_CONFIG_DEPENDS:=CONFIG_LIGHTTPD_SSL
+REBUILD_MODULES=authn_gssapi authn_ldap authn_mysql cml magnet mysql_vhost trigger_b4_dl webdav
+
+PKG_CONFIG_DEPENDS:=CONFIG_LIGHTTPD_SSL $(patsubst %,CONFIG_PACKAGE_lighttpd-mod-%,$(REBUILD_MODULES))
 
 include $(INCLUDE_DIR)/package.mk
 

